hoot-core-test/src/test/cpp/hoot/core/conflate/matching/GreedyConstrainedMatchesTest.cpp

```diff
@@ -54,88 +54,72 @@ using namespace Tgs;
 namespace hoot
 {
 
-class ConstrainedFakeMatch : public Match
+class GreedyConstrainedFakeMatch : public Match
```

Comment:
This was interesting, while debugging this test and the `OptimalConstrainedMatchesTest` the two nearly identical `ConstrainedFakeMatch` classes were being used interchangeably.  It was the weirdest thing I'd ever seen.  So I renamed each one.